salt/metalk8s/salt/master/installed.sls

```diff
@@ -36,13 +36,19 @@ Install and start salt master manifest:
       - file: /etc/salt/master.d/99-metalk8s.conf
       - file: /etc/salt/master.d/99-metalk8s-roots.conf
 
+Delay after new pod deployment:
+  cmd.run:
+    - name: sleep 5
```

Comment:
Consider using `module.wait` with `test.sleep` it's cleaner than doing a cmd.run.
Also as you use `module.wait` you should `watch` the state just above.
(`module.wait` will run only if you have changes in the watching states)

Reply:
should be resolved now